pxr/usd/sdf/mapEditor.cpp

```diff
@@ -178,8 +178,13 @@ Sdf_CreateMapEditor(const SdfSpecHandle& owner, const TfToken& field)
     template std::unique_ptr<Sdf_MapEditor<MapType> >                \
         Sdf_CreateMapEditor(const SdfSpecHandle&, const TfToken&);   \
 
+PXR_NAMESPACE_CLOSE_SCOPE
+
 #include "pxr/base/vt/dictionary.h"
 #include "pxr/usd/sdf/types.h"
+
```

Comment:
Is there a reason that the includes need to be down here?  Could they just be relocated to the top of the header instead of closing/reopening the namespace?

https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#sf4-include-header-files-before-other-declarations-in-a-file

Reply:
Thanks for reviewing this, I believe your suggestion should work out just fine, and I will modify this PR per your suggestion, my original goal was to attempt to keep the changes to this file minimal, but I agree that moving the includes to the top is a much cleaner solution.

Thanks!

Reply:
Modified this PR per your suggestion. 👍🏻